demos/benchmark/lv_demo_benchmark.c

```diff
@@ -591,7 +591,7 @@ static void summary_create(void)
     lv_table_set_cell_value(table, 0, 3, "Avg. time (render + flush)");
 
     /* csv log */
-    LV_LOG("Benchmark Summary (%"LV_PRIu32".%"LV_PRIu32".%"LV_PRIu32" %s)\r\n",
+    LV_LOG("Benchmark Summary (%d.%d.%d %s)\r\n",
```

Comment:
Considering all the other changes are done in the opposite way regarding formatting (general to LVGL style) what was your reasoning here to change the implementation? Just nit-picking, as this works fine, just the other changes are not aligned with this change.

Reply:
Hi @kissa96 
It's not a style thing.
Version number is defined in a plain number, thus %d should be used.
Other modifications use PRIu32 or d32 is because the variable is defined as [u]int32_t. Failing to do so will cause warning on some platform when int != int32_t.

Reply:
Hi @XuNeo, all right, thanks for the clarification!